openssl/src/ssl/mod.rs

```diff
@@ -568,6 +568,7 @@ impl SslContext {
 
         // this is a bit dubious (?)
         try!(ctx.set_mode(ffi::SSL_MODE_AUTO_RETRY));
+        try!(ctx.set_mode(ffi::SSL_MODE_ACCEPT_MOVING_WRITE_BUFFER));
```

Comment:
You can coalesce these into `try!(ctx.set_mode(ffi::SSL_MODE_AUTO_RETRY | ffi::SSL_MODE_ACCEPT_MOVING_WRITE_BUFFER))`.